project_stock/models/stock_move.py

```
@@ -102,6 +102,17 @@ def default_get(self, fields_list):
             )
         return defaults
 
+    def _action_done(self, cancel_backorder=False):
```

Ruff (0.8.2)

Missing return type annotation for private function `_action_done`

(ANN202)